### PR TITLE
Base: Add archive icons and use them for zip files

### PR DESCRIPTION
**Pull request by gigaroby**
Both 16x16 and 32x32 variants have been added.

### CODE REVIEW

File: Base/etc/FileIconProvider.ini [hunk 1]
```diff
@@ -19,3 +19,4 @@ spreadsheet=*.sheets,*.csv
 text=*.txt
 truetype=*.ttf
 pixelpaint=*.pp
+archive=*.zip
```
Comment (bgianfo) [inline Base/etc/FileIconProvider.ini:22]: `*.gz` as well?

Comment (gigaroby) [reply Base/etc/FileIconProvider.ini]: Done, also added *.tar

**Review by bgianfo — APPROVED** [review]

**Comment by metmo** [comment]
@gigaroby I'm looking into some issues related to our PNG Loader, may I ask what application you used to make these icons?

**Comment by gigaroby** [comment]
With pixelpaint inside qemu

**Pull request merged**